src/main/resources/application.conf

```diff
@@ -119,7 +119,7 @@ mantis {
       max-mpt-components-per-message = 200
 
       # Maximum number of peers this node can connect to
-      max-outgoing-peers = 40
+      max-outgoing-peers = 200
```

Comment:
Is this intentional ? I think we can increase it but i would be careful as 200 peers could easily clog the system. Maybe set it to `80` ?